### DISCUSSION TITLE
if i change my username and name after getting github developer pack from another name and username , will it affect on my github student developer pack?

### DISCUSSION THREAD

**Discussion by sijanpaudel14** [community/community#75262] [GitHub Education] [answered]
### Hi everyone,

if i change my username and name after getting github developer pack from another name and username , will it affect on my github student developer pack?

**Comment by elderguardian** [comment]
It should not affect it ([https://github.com/orgs/community/discussions/52888](https://github.com/orgs/community/discussions/52888)).

**Comment by Neeham** [comment, marked as answer]
I've done it in the past and can confirm it doesn't affect it :)
